### DISCUSSION TITLE
Problem with Zigbee stick CC2652R

### DISCUSSION THREAD

**Discussion by steinbock1966** [ct-Open-Source/ct-Smart-Home#64] [Q&A]
I have successfully flashed my Zigbee stick CC2652R and plugged it into my Raspberry Pi4.

When I enter the command ./start.sh start in the terminal I get the following message:
No Texas Instruments USB device found.
No Zigbee adapter found. Not starting Zigbee2MQTT.
Starting the containers
ct-smart-home-master_mqtt_1 is up-to-date
ct-smart-home-master_nodered_1 is up-to-date
After you made yourself familiar with the setup, it's strongly suggested to secure the services. Read the "Security" section in the README!

Why is that, what can I do so that the stick is recognized?

Greetings
Stephan

**Comment by 8xiom** [comment]
Please see https://github.com/ct-Open-Source/ct-Smart-Home/issues/57 and the issues 47 & 52 for hints how to solve this issue. Got my CC2652R working by looking at these.

**Comment by steinbock1966** [comment]
I did 57, 47 and 52, 
unfortunately the Stick is still not recognized!

**Reply by steinbock1966** [reply]
I have successfully flashed my Zigbee stick CC2652R and plugged it into my Raspberry Pi4.
When I enter the command ./start.sh start in the terminal I get the following message:
No Texas Instruments USB device found.

If I enter the command "ls -l / dev / serial / by-id" in the terminal I get the following result:
total 0
lrwxrwxrwx 1 root root 13 Jul 16 12:03 usb-1a86_USB_Serial-if00-port0 -> ../../ttyUSB0

**My configuaration.yaml is:**
Home Assistant integration (MQTT discovery)
homeassistant: false

allow new devices to join
permit_join: true

enable frontend
frontend:
serial:
  port: / dev / ttyUSB0
experimental:
  new_api: true

MQTT settings
mqtt:
  MQTT base topic for zigbee2mqtt MQTT messages
  base_topic: zigbee2mqtt
  MQTT server URL
  server: 'mqtt: // localhost'
  MQTT server authentication, uncomment if required:
  user: my_user
  password: my_password

advanced:
  channel: 25
  network_key: GENERATE
  rtscts: false
  network_key: GENERATE

**My docker-compose.yml is:**
version: '3.6'

services:
  nodered:
    image: ghcr.io/ct-open-source/ctnodered:latest
    volumes:
      - ./data/nodered:/data
      - / etc / localtime: / etc / localtime
    depends_on:
      - mqtt
    restart: always
    network_mode: "host"
    environment:
      - TZ = Europe / Berlin
    
  mqtt:
    image: "eclipse-mosquitto"
    ports:
      - "1883: 1883"
      - "9001: 9001"
    volumes:
      - ./data/mqtt:/mosquitto
      - / etc / localtime: / etc / localtime
    restart: always
    environment:
      - TZ = Europe / Berlin
    
  zigbee:
    image: koenkk / zigbee2mqtt
    volumes:
      - ./data/zigbee:/app/data
      - / run / udev: / run / udev: ro
      - / etc / localtime: / etc / localtime
    devices:
      - "/ dev / ttyACM0: / dev / ttyACM0"
      - "/ dev / ttyACM1: / dev / ttyACM1"
      - "/ dev / ttyUSB0: / dev / ttyUSB0"
    restart: always
    privileged: true
    ports:
      - 1881: 1881
    environment:
      - TZ = Europe / Berlin


**Where is the error that my stick does not work?**

Thanks
Stephan

**Reply by 8xiom** [reply]
You also need to adjust the bash script start.sh because it only recognizes sticks with vendor ID from Texas Instruments. I simply omitted that part in the script (quick & dirty solution), then it worked. See https://github.com/ct-Open-Source/ct-Smart-Home/pull/48 for the details and much more.

**Reply by steinbock1966** [reply]
Can I download the changed start.sh and use it to replace my start.sh?

**Reply by 8xiom** [reply]
> 
> 
> Can I download the changed start.sh and use it to replace my start.sh?

You should have a look at start.sh yourself and adjust it to your needs. For me, just tweaking the if-clause in the start section of start.sh (changing condition "$device == "False"" to whatever condition which is never fulfilled) did th job. Of course this might not work as expected if no stick was found. Use at your own risk!
